package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "colette",
-  "version": "1.0.0-beta",
+  "version": "1.0.0-beta.1",
```

Comment:
:sweat_smile:

Reply:
Fix for beta. I need write some docs ans some missing stores to release V1